wmn-data.json

```diff
@@ -2459,6 +2459,18 @@
         "m_code" : 400,
         "known" : ["johnny", "dev"],
         "cat" : "coding",
+        "valid" : true
+       },
+       {
+        "name" : "Goodgame_Russia",
+        "uri_check" : "hhttps://goodgame.ru/channel/{account}/",
+        "post_body" : "",

```

Comment:
Please remove this line. We changed this to an optional parameter that should only be added when there is POST data.

Reply:
@WebBreacher ✔️